build.go

```diff
@@ -364,6 +364,34 @@ func Build(builder Builder, options ...Option) {
 			return
 		}
 	}
+	logBindings(ctx.Platform.Bindings, logger)
```

Comment:
Curious why we are logging this at the end instead of L#186?

Reply:
I had the logging here initially but in testing, the bindings appeared before the buildpack title & output which seemed confusing. Here it appears at the end of the relevant buildpacks output.